packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -113,10 +133,10 @@ const RichTextEditor = (props: Props) => {
         isInitiallyDisabled={isInitiallyDisabled}
         isEmptyValue={isEmptyValue}
         isEqualValues={deepEquals}>
-        {({ lastRemoteValue, disabled, setValue, externalReset }) => (
+        {({ lastRemoteValue, disabled, setValue }) => (
```

Comment:
Why remove the use of `externalReset`?

Reply:
But I think we still need to support `externalReset` somehow, no?

Reply:
Since the `externalReset changes [only when the remote value changes](https://github.com/contentful/field-editors/blob/48a50159c968dac5e4d623eda1a7f8a43b2ef292/packages/_shared/src/FieldConnector.ts#L98-L123), I think it could be helpful to fix the exact issue this PR is fixing. We [rely on the that in master](https://github.com/contentful/field-editors/blob/48a50159c968dac5e4d623eda1a7f8a43b2ef292/packages/rich-text/src/RichTextEditor.jsx#L269-L270).

The different ist that the old editor didn't have this "store" logic that Plate has. Did you try resetting/destorying Plate store when the `externalReset` changes? something like (pseudo-code):

```javascript
// RichTextEditor
const lastExternalReset = usePreviousValue(externalReset)

useEffect(()=> {
  cosnt oldId = `${getEditorIdFromSDK(sdk)}-${lastExternalReset}`
  clearPlateStore(oldId)
}, [externalReset, ...])

// in the ConnectedRichTextEditor use id= `${getEditorIdFromSDK(sdk)}-${externalReset}`
```